packages/alfa-rules/src/common/diagnostic/contrast.ts

```diff
@@ -102,11 +102,11 @@ export namespace Contrast {
         .writeNumber(this._contrast);
     }
 
-    public get color1(): Color<FirstColor<N>> {
+    public get color1(): Color<Name> {
       return this._color1;
     }
 
-    public get color2(): Color<SecondColor<N>> {
+    public get color2(): Color<Name> {
       return this._color2;
```

Comment:
I needed to revert this bit, because it looks like when the `Color` is tied to `FirstColor<N>` or `<SecondColor<N>>` it allows pairings like `container, background` and the build failed 😨